### PR TITLE
Use polyfill always and trust polyfill.io not to break our application

### PR DESCRIPTION
**Pull request by gjn**
This is a fix for https://github.com/geoadmin/mf-geoadmin3/issues/3237

The polyfill has been updated by hand to include requestAnimationFrame/Array.isArray polyfill for all browsers ('always' flag with 'unknown=polyfill' combination, with guards ('gated' attribute of polyfill request).

[Testlink](https://mf-geoadmin3.dev.bgdi.ch/gjn_poly/)

### CODE REVIEW

File: src/index.mako.html [hunk 1]
```diff
@@ -643,10 +643,10 @@
       window.CESIUM_BASE_URL="${versionslashed}lib/Cesium/";
     </script>
     <script src="${versionslashed}lib/build.js"></script>
+    <script src="${versionslashed}lib/polyfill.min.js"></script>
```
Comment (oterral) [inline src/index.mako.html]: We could include it in build.js

**Comment by gjn** [comment]
@oterral added to build.js. Order was important - which gives me a strange feeling...

File: Makefile [hunk 2]
```diff
@@ -323,7 +322,8 @@ prd/lib/: src/lib/polyfill.min.js \
 	mkdir -p $@
 	cp -rf  $^ $@
 
-prd/lib/build.js: src/lib/jquery.min.js \
+prd/lib/build.js: src/lib/polyfill.min.js \
+			src/lib/jquery.min.js \
```
Comment (oterral) [inline Makefile]: indentation

**Comment by oterral** [comment]
An indentation to fix otherwise lgtm.

**Comment by gjn** [comment]
Thanks alot for the review. As it's the same fix as already on prod, I'll merge this one.

**Pull request merged**